test/Manifest.toml

```diff
@@ -0,0 +1,521 @@
+# This file is machine-generated - editing it directly is not advised
+
+julia_version = "1.11.0-DEV"
+manifest_format = "2.0"
+project_hash = "0d24394284ba951660989e7593688d13df20c8a4"
+
+[[deps.ANSIColoredPrinters]]
```

Comment:
If you remove this manifest and we test this on cuda, this seems reasonable to me.